Cargo.toml

```diff
@@ -30,50 +30,54 @@ deflate = ["zip/deflate"]
 deflate-miniz = ["zip/deflate-miniz"]
 deflate-zlib = ["zip/deflate-zlib"]
 demangle-no-swift = ["symbolic-demangle/cpp", "symbolic-demangle/msvc", "symbolic-demangle/rust"]
-demangle-with-swift = ["symbolic-demangle/cpp", "symbolic-demangle/msvc", "symbolic-demangle/rust", "symbolic-demangle/swift"]
+demangle-with-swift = [
+    "symbolic-demangle/cpp",
+    "symbolic-demangle/msvc",
+    "symbolic-demangle/rust",
+    "symbolic-demangle/swift",
+]
 
 [badges]
 appveyor = { repository = "marco-c/grcov" }
 travis-ci = { repository = "mozilla/grcov" }
 codecov = { repository = "mozilla/grcov" }
 
 [dependencies]
-crossbeam = "^0.8"
-serde = "^1.0"
-serde_derive = "^1.0"
-serde_json = "^1.0"
-walkdir = "^2.3"
-num_cpus = "^1.13"
-semver = "^1.0"
-md-5 = "^0.9"
-zip = { version = "^0.5", default-features = false }
-tempfile = "^3"
-uuid = { version = "^0.8", features = ["v4"] }
-globset = "^0.4"
-quick-xml = "^0.22"
-smallvec = "^1.6"
-rustc-hash = "^1.1"
-clap = "^2.33"
-fomat-macros = "^0.3"
-chrono = { version = "^0.4", features = ["serde"] }
-log = "^0.4"
-simplelog = "^0.10"
-regex = "^1.4"
-rayon = "^1.5"
-cargo-binutils = "^0.3"
-is_executable = "^1.0"
-symbolic-common = "^8.2"
-symbolic-demangle = { version = "^8.1", default-features = false }
-flate2 = "^1.0"
-lazy_static = "^1.3"
-tera = "1"
+cargo-binutils = "0.3"
+chrono = { version = "0.4", features = ["serde"] }
+clap = "2.33"
```

Comment:
We shouldn't use specific versions here but `^`, or it will be harder for people packaging grcov for Linux distributions.

Reply:
This is the same as `^` though, well anything `0.x` / `0.x.x` is the same as `~0.x` / `~0.x.x` and anything `1.x` / `1.x.x` is `^1.x` / `1.x.x` (or bigger numbers than `1`.

For example for `clap` it's no difference, but for `chrono` it technically is `~0.4` now.

Having deps as  for example `^0.3` would be likely to break builds thought doesn't it?

Reply:
Ah I didn't realize that the absence of an operator meant "^": https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio.

Reply:
Oh right, I actually got it a bit wrong too 😅 
So even if we have `0.3` it becomes `^0.3` and not `~0.3`.